BigQuery/src/BigQueryClient.php

```diff
@@ -491,14 +490,15 @@ function (array $job) {
      * ```
      *
      * @param string $id The id of the dataset to request.
+     * @param string|null $projectId The id of the project. **Defaults to** current project id.
```

Comment:
I think more detail on why this option exists would be useful. What does it allow users to do, etc.